src/config.py

```diff
@@ -40,3 +40,7 @@ def is_feature_flag_enabled(flag_name: str) -> bool:
     ).split(",")
     if github_username
 }
+SGTM_FEATURE__STALENESS_CHECK_ENABLED = is_feature_flag_enabled(
+    "SGTM_FEATURE__STALENESS_CHECK_ENABLED"
+)
+SGTM_FEATURE__STALENESS_CHECK_DURATION = int(os.getenv("SGTM_FEATURE__STALENESS_CHECK_DURATION", "0"))
```

Comment:
with 1 flag, we would have to use the github staleness definition by default. That seems fine to me (since it's a generous 14 days) but if you think this behaviour would be acceptable for all SGTM users, I'll consolidate this logic

Reply:
I was thinking more that you use the non-zero presence of this flag instead of where you're using the ENABLED flag. So it'd still gate the feature completely if set to 0 (or unset)